MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -171,6 +171,13 @@ open class RouteProgress: NSObject {
      */
     public var congestionTimesPerStep: [[[CongestionLevel: TimeInterval]]]  = [[[:]]]
 
+    /**
+     An array containing speed limits on the route broken up by leg and then step.
+     */
+    public var maximumSpeedLimitsByLeg: [[[SpeedLimit]]] = []
```

Comment:
Rename to `maximumSpeedLimitsByStep` since the objects are grouped by step. Also rename `speedLimitDistancesByLeg` to `positionedSpeedLimitsByStep` (see below).

Reply:
Did the rename. I removed the `maximumSpeedLimitsByStep` array since I agree it doesn't seem needed now.